Dockerfile-alpine

```diff
@@ -0,0 +1,129 @@
+FROM behance/docker-nginx:5.0-alpine
```

Comment:
note: this is a secondary dockerfile to be leveraged by the Docker Hub build system